atlasdb-client/src/main/java/com/palantir/atlasdb/table/description/render/TableRenderer.java

```diff
@@ -199,7 +199,6 @@ protected void run() {
                 line("package ", packageName, ";");
                 line();
                 importRenderer.renderImports();
-                line();
```

Comment:
It will also fix the issue where there's an extra line when there are no `javax` imports